tests/components/file/conftest.py

```diff
@@ -0,0 +1,34 @@
+"""Test fixtures for file platform."""
+
+from collections.abc import Generator
+from unittest.mock import AsyncMock, MagicMock, patch
+
+import pytest
+
+from homeassistant.core import HomeAssistant
+
+
+@pytest.fixture
+def mock_setup_entry() -> Generator[AsyncMock, None, None]:
+    """Override async_setup_entry."""
+    with patch(
+        "homeassistant.components.file.async_setup_entry", return_value=True
+    ) as mock_setup_entry:
+        yield mock_setup_entry
+
+
+@pytest.fixture
+def is_allowed() -> bool:
+    """Parameterize mock_is_allowed_path, default True."""
+    return True
+
+
+@pytest.fixture
+def mock_is_allowed_path(
```

Comment:
In most other tests we add the path to the allowed dirs set and don't mock the return (as we typically don't want to mock internals. Not sure what `/config` is here so maybe that's not workable?

Reply:
Okay. In this case there are quite some tests that mock `hass.config.is_allowed_path`, so I think it is okay?

Reply:
Yes. Probably better to do in a follow-up anyhow to not mix now. 👍